recipes-debian/netkit-telnet/netkit-telnet_debian.bb

```diff
@@ -0,0 +1,67 @@
+# base recipe: meta-openembedded/meta-networking/recipes-netkit/netkit-telnet/netkit-telnet_0.17.bb
+# base branch: zeus
```

Comment:
warrior branch contains netkit-telnet package, is there any reasons to use zeus branch?